model/api/services.py

```diff
@@ -63,7 +63,7 @@ def _split_text(text: str) -> list[Document]:
     """
     text_splitter = RecursiveCharacterTextSplitter(
         chunk_size=2048,
-        chunk_overlap=100,
+        chunk_overlap=0,
```

Comment:
Are you sure that you want no overlap? It's usually better to have some so that you don't miss contextual clues